build.gradle

```diff
@@ -26,7 +26,31 @@ dependencies {
 
     // Use JUnit test framework
     testImplementation 'junit:junit:4.12'
+    
+    
+    implementation 'com.googlecode.json-simple:json-simple:1.1.1'
+  
+    
 }
+ group 'aws.test'
+ version '1.0'
+
+ apply plugin: 'java'
+
+ sourceCompatibility = 1.8
+
+ repositories {
+   mavenCentral()
+ }
+
+dependencies {
+   implementation platform('org.springframework.boot:spring-boot-dependencies:2.1.0.RELEASE')
+    implementation platform('com.amazonaws:aws-java-sdk-bom:1.11.228')
+    implementation 'com.amazonaws:aws-java-sdk-dynamodb'
+  implementation platform('com.amazonsaws:aws-java-sdk-core:1.11.732')
```

Comment:
Why do you need core for dynamodb, I think in the sandbox commit I used it without core.